### PR TITLE
Infer package name from output directory

### PR DESCRIPTION
**Pull request by jhendrixMSFT**
Removes the need for --namespace.

### CODE REVIEW

File: src/generator/time.ts [hunk 1]
```diff
@@ -21,22 +22,19 @@ export class Content {
...
   }
   return content;
 }
 
-function generateRFC1123Helper(header: string, packageName: string): string {
-  return `${header}
-
-package ${packageName}
+function generateRFC1123Helper(preamble: string): string {
+  return `${preamble}
```
Comment (catalinaperalta) [inline src/generator/time.ts:37]: Semi colon

File: src/generator/time.ts [hunk 2]
```diff
@@ -73,10 +71,8 @@ func (t *timeRFC1123) UnmarshalText(data []byte) error {
 `;
 }
 
-function generateRFC3339Helper(header: string, packageName: string): string {
-  return `${header}
-
-package ${packageName}
+function generateRFC3339Helper(preamble: string): string {
+  return `${preamble}
```
Comment (catalinaperalta) [inline src/generator/time.ts:75]: semicolon

File: src/generator/time.ts (hunk 1 above)
Comment (jhendrixMSFT) [reply src/generator/time.ts:37]: It's at the very end of the text (same with other type).

Comment (catalinaperalta) [reply src/generator/time.ts:37]: Oops you're right I see now😅

**Pull request merged**